MarkeDataService/MarketDataService.Application/Interfaces/IMarketDataService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using MarketDataService.Domain.Entities;
+
+namespace MarketDataService.Application.Interfaces
+{
+    public interface IMarketDataService
+    {
+        Task<List<StockInfoEntity>> GetPopularStocksAsync(string interval);
+       // Task<StockDetailsEntity> GetStockDetailsAsync(string ticker);
```

Comment:
Either uncomment or delete commented code. 
It's a bad practice to keep commented code within the application codebase